notebooks/ai-ml/HuggingFaceEmbeddingGenerationInSpark.ipynb

```diff
@@ -0,0 +1,959 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !pip install numpy==1.26
</pre><p><a href="https://cloud.google.com/dataproc-serverless/docs/concepts/versions/spark-runtime-2.2" target="_blank">https://cloud.google.com/dataproc-serverless/docs/concepts/versions/spark-runtime-2.2</a></p><p>We seem to 1.26 already installed, so not sure why this step is needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>